testnets/symphonytestnet/chain.json

```diff
@@ -33,23 +33,23 @@
   },
   "codebase": {
     "git_repo": "https://github.com/Orchestra-Labs/symphony",
-    "recommended_version": "v23.0.5",
+    "recommended_version": "v0.4.1",
     "compatible_versions": [
-      "v23.0.5"
+      "v0.4.1"
     ],
     "consensus": {
-      "type": "tendermint",
-      "version": "0.37.4"
+      "type": "cometbft",
+      "version": "0.38.11"
     },
     "genesis": {
-      "genesis_url": "https://rpc.testnet.symphonychain.org/genesis"
+      "genesis_url": "https://github.com/Orchestra-Labs/symphony/blob/1cc0ecc1838e65d0cd1eaa6581b69c6237de9efc/networks/symphony-testnet-4/genesis.json"
```

Comment:
This is a link to the Github page/commit for humans. A better solution here is [to link to the raw file](https://raw.githubusercontent.com/Orchestra-Labs/symphony/refs/heads/main/networks/symphony-testnet-4/genesis.json), which can be found by clicking the "Raw" button in the UI. This way, if an app downloads the genesis, it's getting the genesis file minus all of the Github UI (which is what it would get if curl/wgeting the previous link). 

I also switched to the `main` branch for the raw link, which I assume is the correct intention but please change the branch if you think it shouldn't be main.